src/test/java/ProjectTest.java

```diff
@@ -0,0 +1,211 @@
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.litmuschaos.LitmusClient;
+import io.litmuschaos.exception.LitmusApiException;
+import io.litmuschaos.request.CreateProjectRequest;
+import io.litmuschaos.request.LeaveProjectRequest;
+import io.litmuschaos.request.ListProjectRequest;
+import io.litmuschaos.request.ProjectNameRequest;
+import io.litmuschaos.response.CommonResponse;
+import io.litmuschaos.response.ListProjectsResponse;
+import io.litmuschaos.response.ProjectResponse;
+import io.litmuschaos.response.ProjectMemberResponse;
+import io.litmuschaos.response.ProjectRoleResponse;
+import io.litmuschaos.response.ProjectsStatsResponse;
+import io.litmuschaos.response.UserWithProjectResponse;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class ProjectTest {
+
+    private static final String hostUrl = "http://localhost:3000";
+    private static final String username = "admin";
+    private static final String password = "Litmus1234!";
+
+    private static LitmusClient litmusClient;
+    private static String projectId;
+
+    @BeforeAll
+    public static void setup() throws IOException, LitmusApiException {
+        litmusClient = new LitmusClient(hostUrl, username, password);
+        projectId = createProjectAndGetId();
+    }
```

Comment:
It seems like each test in ProjectTest affects the other. I think it would be better for future test code maintenance if you designed each test to work independently of the other. I suggest using @BeforeEach instead of @BeforeAll, and have each test rewrite the given within that test

Reply:
If we use @BeforeEach, I suggest remaining `LitmusClient` initializing logic in @BeforeAll.

Reply:
Oh, thanks for your feedback! But currently, except for the createProject, leaveProject, deleteProject, and updateProjectName methods, all are GET methods. Do you think it would be better to create separate data for the GET methods as well, instead of just creating separate data for those four methods?

Reply:
I think all tests need to be independent each other. So it will be good to create seperate data for all test methods 🙂

Reply:
Checked your modification!